src/EventStore.Common/Configuration/UnitAttribute.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace EventStore.Common.Configuration;
+
+[AttributeUsage(AttributeTargets.Property)]
+public class UnitAttribute(string message) : Attribute {
+	public string Message { get; } = message;
```

Comment:
`Unit` would be a better name for this property

```suggestion
	public string Unit { get; } = unit;
```